docs/content/dependencies.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Dependencies
+description: Dependency Management with Porter
+---
+
+## Define a dependency
+
+In the manifest, list the dependencies in the order that they should be
+installed. Only bundles that have been authored by Porter, and have a Porter
+manifest, can be dependencies.
+
+```yaml
+dependencies:
+- name: mysql
+```
+
+Currently dependencies are assumed to be located in one of the locations below.
+Once the CNAB spec has a mechanism for distributing bundles via registries, this
+will be revisited.
+
+* bundles (in the same directory as the Porter manifest)
+* PORTER_HOME/bundles
+
+Parameters defined in a dependent bundle can be defaulted from the root bundle.
+In the example below, the mysql bundle defined the `database_name` and
+`mysql_user` parameters, and the root bundle (Wordpress) decided to default them
+to a specific value, so that the user wouldn't be required to select a value for
+those parameters when installing Wordpress.
```

Comment:
Is it true to say that parameters defined in dependent bundles which are not defaulted from the root bundle end up being parameters of the resultant composite bundle?

What about parameters defined in dependent bundles which are defaulted from the root bundle? Do these end up being parameters of the resultant composite bundle too, but with default values?

Reply:
It may be worth moving this paragraph into the "Consolidating Parameters and Credentials" section since that section already answers my questions.

Reply:
All parameters defined in the dependencies are merged into the final bundle. I'll reword and maybe move around a bit so that it's introduced in the proper order.